core-parser/src/main/java/org/uniprot/core/parser/tsv/uniparc/UniParcEntryLightValueMapper.java

```diff
@@ -0,0 +1,118 @@
+package org.uniprot.core.parser.tsv.uniparc;
+
+import org.uniprot.core.parser.tsv.EntityValueMapper;
+import org.uniprot.core.uniparc.UniParcEntryLight;
+import org.uniprot.core.uniprotkb.taxonomy.Organism;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class UniParcEntryLightValueMapper implements EntityValueMapper<UniParcEntryLight> {
+
+    private static final List<String> UNIPARC_FIELDS =
+            List.of("upi",
+                    "oldestCrossRefCreated",
+                    "mostRecentCrossRefUpdated",
+                    "organism",
+                    "organism_id",
+                    "gene",
+                    "protein",
+                    "proteome",
+                    "accession",
+                    "first_seen",
```

Comment:
We need to discuss in a Tuesday meeting first_seen X oldestCrossRefCreated attribute. Because "first_seen"  in the EntryParser is used in a difference way. (and this will change for the user)